### DISCUSSION TITLE
Checks shouldn't favor skipped checks unless they're required

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#70544] [Mobile] [answered]
GitHub Mobile beta for Android
https://github.com/firecamp-dev/firecamp/pull/124
It says 4 checks failed but only shows 3. Instead, it shows two skipped checks:
![Screenshot_20231019-073629.png](https://github.com/community/community/assets/2119212/266e58c1-4af3-40d3-b289-6e0d2978ee2b)

The only reason to show skipped checks should be if they're required. Otherwise failed checks should be shown, again, favoring required checks over optional ones.

**Comment by elisealix22** [comment, marked as answer]
@jsoref Once again, thank you for reporting this! I created an issue internally and we will look into it. I appreciate your attention to detail and helping us make the app more useful for you.

**Reply by jsoref** [reply]
Here's today's version
![Screenshot_20241016-121019.png](https://github.com/user-attachments/assets/8424158e-b413-4d7a-b419-6f2c827fa855)

5 checks failed, but none are shown... I'm sure they're somewhere, but I definitely wanted to see them as opposed to the skipped checks.
